src/toady/resolve_service.py

```
@@ -0,0 +1,223 @@
+            thread_id: The thread ID that caused the error.
+            action: The action being performed (resolve/unresolve).
+
+        Raises:
+            ThreadNotFoundError: If thread is not found.
+            ThreadPermissionError: If permission is denied.
+            ResolveServiceError: For other GraphQL errors.
+        """
+        error_messages = []
+        for error in errors:
+            message = error.get("message", str(error))
+            error_type = error.get("type", "")
```

Ruff (0.11.9)

Local variable `error_type` is assigned to but never used

Remove assignment to unused variable `error_type`

(F841)